core/test/junit/saros/session/SarosSessionUserTest.java

```diff
@@ -0,0 +1,133 @@
+package saros.session;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import java.util.HashMap;
+import java.util.List;
+import org.junit.After;
+import org.junit.Before;
+
+import org.junit.Test;
+import saros.net.xmpp.JID;
+import saros.session.User;
+import saros.session.UserPrivilege;
+
+public class SarosSessionUserTest {
+
+  private JID ALICE = new JID("alice@test/Saros");
+  private JID BOB = new JID("bob@test/Saros");
+
+  private User HOST = new User(ALICE, true, true, 0, 0);
+  private User INVITEE = new User(BOB, true, true, 0, 0);
+
+  protected void setUp() {
+	  // System.out.println("1 - SarosSessionUserTest setUp()");
+  }
+  
+  // testing default privileges, all should return false
+  @Test
+  public void testSessionUserReadOnlyPrivilegeDefault() {
+	  setUp();
+// System.out.println("2 - testSessionUserReadOnlyPrivilegeDefault()" + HOST.hasReadOnlyAccessPrivilege());
+	  assertEquals(HOST.hasReadOnlyAccessPrivilege(), new Boolean(false));
+  }
+  @Test
+  public void testSessionUserWritePrivilegeDefault() {
+	  assertEquals(HOST.hasWriteAccessPrivilege(), new Boolean(false));
+  }
+  @Test
+  public void testSessionUserShareDocumentPrivilegeDefault() {
+	  assertEquals(HOST.hasShareDocumentPrivilege(), new Boolean(false));
+  }
+  @Test
+  public void testSessionUserInvitePrivilegeDefault() {
+	  assertEquals(HOST.hasInvitePrivilege(), new Boolean(false));
+  }
+  @Test
+  public void testSessionUserGrantPermissionPrivilegeDefault() {
+	  assertEquals(HOST.hasGrantPermissionPrivilege(), new Boolean(false));
+  }
+  @Test
+  public void testSessionUserJoinSessionPrivilegeDefault() {
+	  assertEquals(HOST.hasJoinSessionPrivilege(), new Boolean(false));
+  }
+  @Test
+  public void testSessionUserStartSessionPrivilegeDefault() {
+	  assertEquals(HOST.hasStartSessionServerPrivilege(), new Boolean(false));
+  }
+  @Test
+  public void testSessionUserStopSessionPrivilegeDefault() {
+	  assertEquals(HOST.hasStopSessionServerPrivilege(), new Boolean(false));
+  }
+  @Test
+  public void testSessionUserDeleteSessionDataPrivilegeDefault() {
+	  assertEquals(HOST.hasDeleteSessionDataPrivilege(), new Boolean(false));
+  }
+  @Test
+  public void testSessionUserConfigureServerPrivilegeDefault() {
+	  assertEquals(HOST.hasConfigureServerPrivilege(), new Boolean(false));
+  }
+
+  // test add privilege
```

Comment:
While you have add methods, you do not have any remove method but only a generic set method. So what about testing this behavior ?